build.gradle.kts

```diff
@@ -45,6 +45,7 @@ val javaPlugin = "java"
 val javaScriptPlugin = "JavaScript"
 // BACKCOMPAT: 2020.2
 val clionPlugins = if (platformVersion < 203) emptyList() else listOf("com.intellij.cidr.base", "com.intellij.clion")
+val mlCompletionPlugin = "com.intellij.completion.ml.ranking"
```

Comment:
Looks like `com.intellij.completion.ml.ranking` was introduced only in 2020.3 (extracted from `com.intellij.stats.completion`) so current impls cannot be applied to 2020.2

I see 2 variants:
- try to support 2020.2 as well. It leads to a lot of conditions in build scripts and xmls
- support only 2020.3

I prefer the second option (support only 2020.3) because it's simpler (for example, `CatBoostJarCompletionModelProvider` exists only in 2020.3) and we are going to drop 2020.2 support quite soon

Reply:
I agree with you, so I moved all the code into `203` folder